src/views/TermsAndConditions/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import Header from 'components/Header';
+import MainContent from 'components/MainContent';
+
+const TermsAndConditions = () => {
+  const { t } = useTranslation();
```

Comment:
If we pass a top-level key in the initial call to `useTranslation` like this:

```suggestion
  const { t } = useTranslation('termsAndConditions');
```

It means that when we call `t` we can use a shorter key that is also more scannable for humans:

```jsx
<p className="margin-y-4">{t('unauthorizedUse')}</p>
```

Reply:
Good catch. I'll make this change.